ui/client/components/graph/TypeSuggester.js

```diff
@@ -0,0 +1,19 @@
+import _ from 'lodash';
+
+export default class TypeSuggester {
+
+  constructor(typesInformation){
+    this._typesInformation = typesInformation;
+    console.log("Types info" + typesInformation)
```

Comment:
Please remove console.log
It should not be in production code

Reply:
done